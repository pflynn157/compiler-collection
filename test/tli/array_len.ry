
func main -> i32 is
    array a1 : i32[10];
    array a2 : string[5];
    array a3 : i8[100];
    
    print(length(a1));
    print(length(a2));
    print(length(a3));
    
    return 0;
end

