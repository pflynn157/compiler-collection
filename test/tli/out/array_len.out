10
5
100
